### ISSUE TITLE
Let's Do good with Data !

### ISSUE THREAD

**Issue by mridubhatnagar** [pybites/challenges#162]
Take any available open public dataset.
And draw insights out of it. 

Public Data Example:
1) List of primary schools data, something related to education
2) Election data etc..

Cleaning, processing, cool data visualizations

**Comment by bbelderbos** [comment]
yep we can use more of these, so we had a ‘defined dataset’ one (marvel / PCC44), this would be an open/free one?

**Comment by mridubhatnagar** [comment]
Yeah. I thought so. Those are huge datasets I believe. 
Can be turned useful if we come up with some interesting thing out of it.

**Comment by mridubhatnagar** [comment]
In marvel from where were we getting the dataset? 
Did you scrape something up for getting that?

**Comment by bbelderbos** [comment]
could leave people free to it, maybe do ask them to use Pandas (?)
the Marvel csv I found on Github

**Comment by mridubhatnagar** [comment]
Yes, that can be done. But specifying even the module to use. Won't that be too restricted?

**Comment by bbelderbos** [comment]
true, so let's do this one as an open data analysis challenge, and have another one dedicated to pandas :)

**Comment by mridubhatnagar** [comment]
cool. Sounds good to me.

**Comment by mridubhatnagar** [comment]
some public data sets - https://cloud.google.com/bigquery/public-data/#usa-names

**Comment by bbelderbos** [comment]
What works best in our experience is to point to a specific dataset and define the questions challenge takers need to answer. See for example PCC51 we launched today. Do you have any specific set in mind we can use? Happy to collaborate with you ...
